### DISCUSSION TITLE
Why enforce certain naming conventions?

### DISCUSSION THREAD

**Discussion by xoldd** [graphql-hive/graphql-eslint#1464] [Q&A]
@B2o5T Naming convention enforced by the `naming-convention` rule, is that a graphql standard? Or only cuz eslint is javascript specific so you're following javascript's naming convention?

**Comment by dimaMachina** [comment]
There is no standard for it but it follows best practices from https://graphql-rules.com/rules/naming

**Comment by xoldd** [comment]
@B2o5T Do graphql implementations in other languages like rust, golang etc follow this convention too, like whatever their linting tools are for this.

**Reply by dimaMachina** [reply]
idk
